components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/policy/store/DefaultPolicyDataStore.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.entitlement.policy.store;
+
+import org.wso2.balana.combine.PolicyCombiningAlgorithm;
+import org.wso2.carbon.identity.entitlement.EntitlementException;
+import org.wso2.carbon.identity.entitlement.dto.PolicyStoreDTO;
+
+import java.util.Properties;
+
+/**
+ * This is the default implementation of PolicyDataStore
+ */
+public class DefaultPolicyDataStore implements PolicyDataStore {
+
+    @Override
+    public void init(Properties properties) throws EntitlementException {
+
+    }
+
+    @Override
+    public PolicyCombiningAlgorithm getGlobalPolicyAlgorithm() {
+
+        return null;
+    }
+
+    @Override
+    public void setGlobalPolicyAlgorithm(String policyCombiningAlgorithm) throws EntitlementException {
+
+    }
+
+    @Override
+    public String getGlobalPolicyAlgorithmName() {
+
+        return null;
+    }
+
+    @Override
+    public String[] getAllGlobalPolicyAlgorithmNames() {
+
+        return new String[0];
+    }
+
+    @Override
+    public void setPolicyData(String policyId, PolicyStoreDTO policyDataDTO) throws EntitlementException {
+
```

Comment:
Let's add a comment indicating why this is not executing anything.

ex: Policy data already stored with xyz